UPGRADE.md

```diff
@@ -114,13 +114,21 @@ Removed following Classes:
 
 * Sulu\Bundle\CoreBundle\Build\CacheBuilder
 
+Removed following services:
+
+* `sulu_contact.account_repository` (use `sulu.repository.account`)
+* `sulu_contact.contact_repository` (use `sulu.repository.contact`)
+
 Renamed following Methods:
 
 * Sulu\Bundle\MediaBundle\Entity\CollectionRepositoryInterface::count => Sulu\Bundle\MediaBundle\Entity\CollectionRepositoryInterface::countCollections
 
-Rename following classes:
+Rename following classes/interfaces:
 
 * Sulu\Component\HttpCache\HttpCache => Sulu\Bundle\HttpCacheBundle\Cache\AbstractHttpCache
+* Sulu\Component\Contact\Model\ContactInterface => Sulu\Bundle\ContactBundle\Model\ContactInterface
```

Comment:
Although I generally like the `Model` namespace better than `Entity` I would still go for `Entity` because it would be more consistent (or do we already have the `Model` namespace in any bundle?)

Reply:
Didn't you adjust the upgrade file?